src/dialog-editor/components/toolbox/toolboxComponent.ts

```diff
@@ -0,0 +1,132 @@
+class DialogField {
+  public icon: string;
+  public label: string;
+  public placeholders: any;
+
+  constructor(type: string,
+              icon: string,
+              label: string,
+              options: any = {}) {
+    this.icon = icon;
+    this.label = label;
+    this.placeholders = Object.assign({
```

Comment:
I agree, but these are trivial things. But ultimately need to migrate their way to tslint or whatever. If we want them to become idiomatic for MiQ.